lxd/containers.go

```diff
@@ -2165,6 +2167,8 @@ func containerExecPost(d *Daemon, r *http.Request) Response {
 	if post.WaitForWS {
 		ws := &execWs{}
 		ws.fds = map[int]string{}
+		ws.rootUid = int(d.idMap.Uidmin)
```

Comment:
I think we do right now (until we fix #632), because Uidmin will always be whatever the user has it set to, regardless of whether or not the container is privileged.